<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="BSLGDC-L2045" name="nbcu.compass.amorttemplate.test.AmortTest" tests="1" failures="1" timestamp="09 Feb 2018 10:04:25 GMT" time="69.469" errors="0">
  <testcase name="testAmortTemplateUS" time="69.469" classname="nbcu.compass.amorttemplate.test.AmortTest">
    <failure type="java.lang.AssertionError" message="An unknown error occurred in the remote end while processing the command. (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 390 milliseconds
Build info: version: &#039;3.7.1&#039;, revision: &#039;8a0099a&#039;, time: &#039;2017-11-06T21:01:39.354Z&#039;
System info: host: &#039;BSLGDC-L2045&#039;, ip: &#039;100.122.39.127&#039;, os.name: &#039;Windows 7&#039;, os.arch: &#039;x86&#039;, os.version: &#039;6.1&#039;, java.version: &#039;1.8.0_131&#039;
Driver info: io.appium.java_client.windows.WindowsDriver
Capabilities [{platformName=Windows, appTopLevelWindow=2b1092, platform=ANY}]
Session ID: A0D899F4-DF88-411E-BF0C-E7CA455DCB18">
      <![CDATA[java.lang.AssertionError: An unknown error occurred in the remote end while processing the command. (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 390 milliseconds
Build info: version: '3.7.1', revision: '8a0099a', time: '2017-11-06T21:01:39.354Z'
System info: host: 'BSLGDC-L2045', ip: '100.122.39.127', os.name: 'Windows 7', os.arch: 'x86', os.version: '6.1', java.version: '1.8.0_131'
Driver info: io.appium.java_client.windows.WindowsDriver
Capabilities [{platformName=Windows, appTopLevelWindow=2b1092, platform=ANY}]
Session ID: A0D899F4-DF88-411E-BF0C-E7CA455DCB18
	at org.testng.Assert.fail(Assert.java:93)
	at nbcu.compass.amorttemplate.util.Log.fail(Log.java:420)
	at nbcu.compass.amorttemplate.test.AmortTest.testAmortTemplateUS(AmortTest.java:101)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- testAmortTemplateUS -->
</testsuite> <!-- nbcu.compass.amorttemplate.test.AmortTest -->
